spec/api/v2/management/users_spec.rb

```diff
@@ -121,17 +121,10 @@
       end
 
       context 'users' do
-        let(:test_user) { create(:user, email: 'testa@gmail.com', role: 'admin') }
-        let(:second_user) { create(:user, email: 'testb@gmail.com') }
-        let(:third_user) { create(:user, email: 'testd@gmail.com') }
-        let(:fourth_user) { create(:user, email: 'testc@gmail.com') }
-
-        before(:example) {
-          test_user
-          second_user
-          third_user
-          fourth_user
-        }
+        let!(:test_user) { create(:user, email: 'testa@gmail.com', role: 'admin') }
```

Comment:
do we need this users?

Reply:
yes, as soon as we testing pagination, it will be a good test to have
```page 1 limit 2
page 2 limit 2```

Also those users used by already created tests in "user context" (please check whole file)